tests/conftest.py

```
@@ -141,6 +120,17 @@ def pytest_configure(config):
         print(f"\nUsing cached webpack script from {CACHE_FILE}")
 
 
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item, call):
```

Ruff (0.14.13)

Unused function argument: `call`

(ARG001)